util/gh-pages/index_template.html

```diff
@@ -24,6 +24,7 @@
     <link id="styleNight" rel="stylesheet" href="https://rust-lang.github.io/mdBook/tomorrow-night.css" disabled="true"> {# #}
     <link id="styleAyu" rel="stylesheet" href="https://rust-lang.github.io/mdBook/ayu-highlight.css" disabled="true"> {# #}
     <link rel="stylesheet" href="style.css"> {# #}
+    <noscript><link rel="stylesheet" href="noscript.css"></noscript> {# #}
```

Comment:
We could avoid needing a new CSS file by using the `<html class="noscript">` pattern,  having `theme.js` remove the class

Reply:
That makes the CSS bigger for everyone but as you prefer.